WcaOnRails/config/locales/en.yml

```diff
@@ -1180,10 +1180,11 @@ en:
     mission:
       title: "Mission"
       paragraphs:
-        '1': "The World Cube Association governs competitions for all puzzles labelled as Rubik puzzles, and all other puzzles that are played by twisting the sides, so-called 'twisty puzzles'. Most famous of these puzzles is the Rubik's Cube, invented by professor Rubik from Hungary. A selection of these puzzles are chosen as official events of the WCA."
+        '1': "The World Cube Association governs competitions for mechanical puzzles that are operated by twisting groups of pieces, commonly known as 'twisty puzzles'. Most famous of these puzzles is the Rubik's Cube, invented by professor Rubik from Hungary. A selection of these puzzles are chosen as official events of the WCA."
         '2': "The World Cube Association organizes competitions all over the world, supported by regional organizations responsible for organizing competitions in their countries."
-        '3': "The goal of the World Cube Association is to have<br /><strong>more competitions in more countries with more people and more fun, under fair and equal conditions.</strong>"
-        '4': "The spirit of the World Cube Association is that<br />
+        '3': "The Mission of the World Cube Association is to have<br/>
```

Comment:
This feels a little strange to me because we're already in a section titled "Mission", so what were paragraphs 1 and 2 all about?

Reply:
Yeah, it's been like this for God knows how long...

It would make sense to remove that paragraph title, maybe.

Reply:
It makes sense to me too to get rid of the "Mission" title. And besides, what about the `<br />` preceding the `<strong>`? Couldn't we remove those too?